sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -2259,8 +2259,12 @@ public void cancelHealthcareLro(HttpClient httpClient, TextAnalyticsServiceVersi
                 syncPoller = client.beginAnalyzeHealthcareEntities(documents, options).getSyncPoller();
             syncPoller = setPollInterval(syncPoller);
             syncPoller.cancelOperation();
+            LongRunningOperationStatus operationStatus = syncPoller.poll().getStatus();
+            while (!LongRunningOperationStatus.USER_CANCELLED.equals(operationStatus)) {
```

Comment:
I'm not familiar with the java tests, but I assume we would want some sort of test timeout here, should the `getStatus()` call fail to be cancelled.